cmd/auth-service/main.go

```diff
@@ -31,13 +34,18 @@ func main() {
 	flag.StringVar(&keyFile, "keyFile", "/certs/key.pem", "Path to key file")
 	flag.BoolVar(&useTLS, "useTls", false, "Enable HTTPS server")
 
+	flag.StringVar(&awsConfig.AwsAccessKeyID, "awsAccessKeyId", "", "AWS IAM AccessKeyID for the Liqo User")
```

Comment:
Personally, I would the AWS credentials in a dedicated secret object.

Reply:
fixed in the #755